cmd/ltcoordinator/main.go

```diff
@@ -16,7 +16,8 @@ import (
 )
 
 func RunCoordinatorCmdF(cmd *cobra.Command, args []string) error {
-	cfg, err := coordinator.GetConfig()
+	configFilePath, _ := cmd.Flags().GetString("config")
```

Comment:
Nit: I know we are ignoring this everywhere, and I realize it does not matter anyways. But I am a bit torn on what to do about this. Any thoughts ?

Reply:
I actually thought it would return error if the config flag was not defined but doesn't look like it so I guess we can handle it as any other error.

Reply:
I suggest to drop `cobra` and `viper` and just use `flag`and a couple of `switch`es. I am not sure if we are getting enough efficiency from these dependencies.

Reply:
I am always up for removing dependencies. Mainly I tried to follow what the old load-test was doing, which was trying to follow what the main server app was doing. 
We can definitely make a ticket to address this if you think it won't complicate things.

Reply:
I think flag is useful when there is only one level of flags. But when we have subcommands, it takes a lot of boilerplate to get it working. In those cases, cobra is kinda useful.